tests/pytest/test_mem.py

```diff
@@ -0,0 +1,79 @@
+
+from common import *
+import time
+import datetime
+
+
+CHECK_MEMRECLAIM = os.getenv('MEMRECLAIM', '0') == '1'
+SERDE_JSON = os.getenv('SERDE_JSON', '0') == '1'
+
+JSON_FILES = [
+    {'file': 'https://raw.githubusercontent.com/mloskot/json_benchmark/master/data/canada.json',
+     'vsz': 2880000 if SERDE_JSON else 598000},
+    {'file': 'https://raw.githubusercontent.com/RichardHightower/json-parsers-benchmark/master/data/citm_catalog.json',
+     'vsz': 847000 if SERDE_JSON else 200000},
```

Comment:
Why not run the tests both with serde_json and ijson?

Reply:
but they do. when we run with serde_json, `SERDE_JSON` env var is 1.

Reply:
I meant why you want to control it from outside and not just run on both here?

Reply:
There's a full cycle of tests running with `JSON_BACKEND SERDE_JSON`, and when this is the case, we'll be testing it's memory usage. Otherwise, we'll be doing both backends on each cycle, which is redundant and somewhat confusing.